e2e-tests/demand-backup/conf/restore-on-demand-backup-azure-blob.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: pxc.percona.com/v1
+kind: PerconaXtraDBClusterRestore
+metadata:
+  name: on-demand-backup-azure-blob
+spec:
+  pxcCluster: demand-backup
+  backupName: on-demand-backup-azure-blob
```

Comment:
Could we (maybe in a separate task) use the same field **clusterName** instead of pxcCluster as we use in psmdb?